http_cache.rst

```diff
@@ -157,6 +157,32 @@ When you're in debug mode (the second argument of ``Kernel`` constructor in the
 front controller is ``true``), Symfony automatically adds an ``X-Symfony-Cache``
 header to the response. Use this to get information about cache hits and misses.
 
+.. tip::
```

Comment:
I am against such a “big” tip, why not adding a new dedicated section with a headline?